### DISCUSSION TITLE
Set status color based on sensors state value

### DISCUSSION THREAD

**Discussion by fernandolcardoso** [home-assistant/frontend#22560] [Other feature requests]
I'm looking for a suggestion that is not through adjusting the interface, but through yaml definitions

![image](https://github.com/user-attachments/assets/936aff2d-22d9-4bc2-9564-5451485b03e9)

_In the image above, I am getting data through RESTful Sensor (rest:) and I have included some lines as a suggestion_

The idea is to facilitate the inclusion of 'Tile cards' with pre-defined colors when obtaining data, visually facilitating the importance of the information.

Much like choosing the color of a light bulb that is easily visible on the Dashboard card.

I believe that color can represent a very important visual identity when viewed.

For example, I would like to monitor the level of several rivers in my city and region, and depending on the level, a color would be represented by its level of importance, or defined by me, such as a red color, would represent the level of a river would be exceeding the limits of the edge.

Thank you for your attention and I hope you consider this request.

**Comment by marcinbauer-ohf** [comment]
This is a good idea, but before this we would need to clean up the current usage of colors in the cards since they are used mainly for differentiating domain types, light hue, and climate status. After this a sensor type card could have (optional) different colors for different ranges.

**Reply by fernandolcardoso** [reply]
Hello @marcinbauer85,

Thank you for your collaboration, the idea here would be more for sensor situations, there would be no need to manipulate the colors of the lights, just relevant information as in the example I mentioned above.

I believe that colors have a great influence on visual interpretation, we don't always attach numbers to their due importance.

I hope that the Home Assistant team takes this special request into consideration so that we can use the system's native resources.

Remembering that this feature, in my opinion, would be great in all aspects, such as when viewing more details, or when adding a block without the need for external customization.

**Comment by oerix** [comment]
Any update on this please? It will be a very welcomed addition.
